README.md

```diff
@@ -1,75 +1,75 @@
 # React component library for Directories project
 ![Travis (.org)](https://img.shields.io/travis/moneyadviceservice/react_library)
 
-**List of the technologies used in this library:**
+## Documentation
+Visit the [documentation website](https://moneyadviceservice.github.io/react_library/) for more information.
+
+## List of the technologies used in this library:
 
 - [React](https://reactjs.org/) for creating components and [styled-components](https://www.styled-components.com/) for styling;
 - [Styleguidist](https://react-styleguidist.js.org/) for both live development and generating documentation;
 - [Jest](https://jestjs.io/) with [React Testing Library](https://testing-library.com/docs/react-testing-library/intro) for testing;
 - [Rollup](https://rollupjs.org/guide/en/) and [Babel](https://babeljs.io/) for bundling the library for publishing to [NPM](https://www.npmjs.com/);
 - [Travis CI](https://travis-ci.org/) for auto-deployment of the documentation to GitHub pages.
 
-## Usage
-
-> ### Installation
-
-- `$ npm i @moneypensionservice/directories`
-
-> ### Usage
-
-- `import { Container, Row, Col } from '@moneypensionservice/directories'`
-
-[Check the documentation](https://moneyadviceservice.github.io/react_library/) for more examples.
-
-
-## Development Scripts
-
-> ### Install Dependencies
-
-- `$ npm install`
-
-
-> ### Preview Components
-
-- `$ npm start`
-  - _live reloading and preview of the components at `http://localhost:6060`_
-
-
-> ### Testing
-
-- `$ npm test`
-
-
-> ### Build for Production
-
-**Note:** Update the package version in [`package.json`](https://github.com/moneyadviceservice/react_library/package.json) before building
-
-- `$ npm run prepublishOnly`
-  - _cleanup and rebuild the files in the `lib` directory_
-- `$ npm run postbuild`
-  - _shows the contents of the package that will be published to NPM_
-
-
-> ### Publish Package to NPM
+## Installation
+You can install the component library using either of the methods given below.
+
+For npm users:
+```shell
+  $ npm install @moneypensionservice/directories
+```
+For Yarn users:
+
+```shell
+  $ yarn add @moneypensionservice/directories
+```
+
+# Development Scripts
```

Comment:
It would be useful here to explain to Devs without background knowledge that this section is about running the library standalone?